src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const operate = (callback, num) => {
+    result = callback(result, num);
+
+    return calculator;
+  };
+
+  const add = (prevResult, num) => {
+    return prevResult + num;
+  };
+
+  const subtract = (prevResult, num) => {
+    return prevResult - num;
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator `=` to subtract something from the existing value. Instead of returning `prevResult - num`, you can use `-=` to update `result` directly.